dockerfiles/init/manifests/che.env

```diff
@@ -116,12 +116,6 @@
 #     Example: che.docker.securityopt=seccomp:unconfined,apparmor:unconfined
 #CHE_DOCKER_SECURITYOPT=seccomp:unconfined
 
-# Agent Start Timeout
-#     The length of time that a workspace will be allowed to boot before the system terminates the
-#     boot process. If the Che container cannot establish two way communications with the
-#     agents within the workspace when it boots, then the workspace will not be started.
-#CHE_WORKSPACE_AGENT_DEV_MAX__START__TIME__MS=300000
```

Comment:
similar question what is the reason for removing this?

Reply:
@mkuznyetsov didn't found any code that is using it?

Reply:
this property has never been used in Che 6, it was used in WsAgentLauncher.class in Che 5.

I guess it can be removed from here as well, for consistency
https://github.com/eclipse/che-docs/blob/master/src/main/pages/setup-docker/docker-config.md